Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.h

```diff
@@ -0,0 +1,120 @@
+#ifndef CGAL_SCENE_SURFACE_MESH_ITEM_H
+#define CGAL_SCENE_SURFACE_MESH_ITEM_H
+//Defines the precision of the positions (for performance/precision sake)
+#define GL_DATA GL_FLOAT
+#define gl_data float
+#define IS_FLOAT 1
+
+#include "Scene_surface_mesh_item_config.h"
+#include <CGAL/Three/Scene_item.h>
+#include <CGAL/Three/Viewer_interface.h>
+#include <vector>
+
+#include <boost/scoped_ptr.hpp>
+#include <boost/array.hpp>
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Surface_mesh/Surface_mesh_fwd.h>
+#include <CGAL/Surface_mesh/Surface_mesh.h>
```

Comment:
Non-sense! If that file does not need the definition of the `Surface_mesh` then it should include the `fwd` header only, and let the `.cpp` file include the other header.